pkg_template.html

```diff
@@ -69,11 +69,9 @@
         <b>
           Project links :
         </b>
-        <a href="_homepage" target="_blank" id="repoHomepage">
-          <button>
-            Github Repository
-          </button>
-        </a>
+        <button onclick="location.href='_homepage'" id="repoHomepage">
```

Comment:
can we add that the homepage opens in a new tab?

Reply:
In the official Pypi index, it doesn't open in a new page, I think it would be better to follow the same behavior as the official Pypi index

Reply:
This page https://components.espressif.com/ is also a index they open a new tab, is the one that i took as a reference.

Also that change on opening the page on a new tab was something that many devs ask me to add on the company where i work, so... i don't know if add it here too?

Reply:
Actually I had 0 feedback on this, so if you tell me several people ask you this feature.... Let's add it, that seems a good idea to follow users !

But I have no idea how to do that without using an anchor... Any idea ?

Reply:
i will search on how to do it